.github/workflows/lte-integ-test-bazel.yml

```diff
@@ -78,7 +78,9 @@ jobs:
         run: |
           cd lte/gateway
           vagrant ssh -c 'cd ~/magma; bazel/scripts/remote_cache_bazelrc_setup.sh "${{ env.CACHE_KEY }}" "${{ env.REMOTE_DOWNLOAD_OPTIMIZATION }}" "${{ secrets.BAZEL_REMOTE_PASSWORD }}";' magma
-          vagrant ssh -c 'cd ~/magma; bazel build --profile=bazel_profile_lte_integ_tests `bazel query "kind(.*_binary, //orc8r/... union //lte/... union //feg/...)"`;' magma
+          REGEX='"service|util_script"'
+          QUERY="'attr(tags, "${REGEX}", kind(.*_binary, //orc8r/... union //lte/... union //feg/... except //lte/gateway/c/core:mme_oai))'"
+          vagrant ssh -c 'cd ~/magma; bazel build --profile=bazel_profile_lte_integ_tests `bazel query '"${QUERY}"'`;' magma
```

Comment:
With QUERY outsourced to a variable I find it harder to understand where we need to sprinkle some quotes to get the escaping right

Reply:
In general I find this escaping and quoting business quite ugly and would  prefer to just define one Bazel target `lte_agw` that depends on all necessary services and scripts. (out of scope for this PR though)

Reply:
This is how it would look like with a oneliner:
```
vagrant ssh -c 'cd ~/magma; bazel build --profile=bazel_profile_lte_integ_tests `bazel query '\''attr(tags, "service|util_script", kind(.*_binary, //... except //lte/gateway/c/core:mme_oai))'\''`;' magma
```
For me this looks less readable, but neither of them are good.

The point of querying is that the targets can change over time, a static build target would add additional maintenance.

Reply:
When trying the query, I got
```
> REGEX='"service|util_script"'
> QUERY="'attr(tags, "${REGEX}", kind(.*_binary, //orc8r/... union //lte/... union //feg/... except //lte/gateway/c/core:mme_oai))'"
> vagrant ssh -c 'cd ~/magma; echo `bazel query '"${QUERY}"'`;' magma

Loading: 0 packages loaded
INFO: Empty results
Loading: 0 packages loaded
```
I tried this locally, not on a runner. Do the same commands work for you locally?

Reply:
> a static build target would add additional maintenance.

I think it wouldn't be a lot of maintenance and probably more easy to understand than the dynamic query. We are even already maintaining such a list for the `magma_deb` and `sctpd_deb` targets. Not sure what the overhead is to build the packages, maybe we could even do a `bazel build lte/gateway/release:{magma,sctpd}_deb_pkg` here.

Reply:
> > a static build target would add additional maintenance.
> 
> I think it wouldn't be a lot of maintenance and probably easier to understand than the dynamic query. We are even already maintaining such a list for the `magma_deb_pkg` and `sctpd_deb_pkg` targets. Not sure what the overhead is to build the packages, maybe we could even do a `bazel build lte/gateway/release:{magma,sctpd}_deb_pkg` here.

The packages are already being built in a separate job in `bazel.yml`. 

I would also prefer the query over a static target - but we can discuss that if needed.